programs/cleos/main.cpp

```diff
@@ -3321,16 +3327,26 @@ int main( int argc, char** argv ) {
          chain_id = chain_id_type(str_chain_id);
       }
 
-      if( str_private_key.size() == 0 ) {
+      if( str_private_key.size() > 0 ) {
+         private_key_type priv_key;
+         try {
+            priv_key = private_key_type(str_private_key);
+         } EOS_RETHROW_EXCEPTIONS(private_key_type_exception, "Invalid private key: ${private_key}", ("private_key", str_private_key))
```

Comment:
Don't echo the private key.

Reply:
PR updated. No longer echoing invalid private key